packages/catalog-process/src/services/readModelService.ts

```diff
@@ -398,20 +398,40 @@ export function readModelServiceBuilder(
         ),
       };
     },
-    async getEServiceByNameAndProducerId({
+    async existsEServiceWithNameAndProducerId({
```

Comment:
See this comment left on the PR for eservice template names: https://github.com/pagopa/interop-be-monorepo/pull/1970/files#r2135766548

If you address it there, I would also align naming of these two methods